public/lib/obs/obs_classes.py

```diff
@@ -222,18 +232,19 @@ def verify_all(self):
                 obs_chapter = OBSChapter(chapter)
             errors = errors + obs_chapter.get_errors()
 
-        if len(errors) == 0:
-            print('No errors were found in the OBS data.')
-            return True
-        else:
-            return False
+        if errors: return False
+        # else:
+        print('No errors were found in the OBS data.')
+        return True
+
 
 
 class OBSEncoder(JSONEncoder):
     def default(self, o):
         return o.__dict__
 
 
+
```

Comment:
PyCharm/IntelliJ complains loudly about the extra white space.
![image](https://user-images.githubusercontent.com/5262964/61294366-a5855f80-a7a3-11e9-8040-c6b879e87ecd.png)